src/controller/datacontroller.js

```diff
@@ -143,17 +145,18 @@ export default class DataController {
 		const modelRange = ModelRange.createIn( modelElementOrFragment );
 
 		const viewDocumentFragment = new ViewDocumentFragment();
+		const viewWriter = new ViewWriter( new ViewDocument() );
 		this.mapper.bindElements( modelElementOrFragment, viewDocumentFragment );
 
-		this.downcastDispatcher.convertInsert( modelRange );
+		this.downcastDispatcher.convertInsert( modelRange, viewWriter );
```

Comment:
We should make it symmetrical. Since `downcastDispatcher` do not have `view` as a property and `change` is called outside, `upcastDispatcher` may have no `model` and also get `writer` as a parameter.